
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
        <meta name="description" content="Documentation for the CropXR project">
      
      
        <meta name="author" content="CropXR">
      
      
      
        <link rel="prev" href="../defining-a-study/">
      
      
        <link rel="next" href="../../policies/data-sensitivity-levels/">
      
      
        
      
      
      <link rel="icon" href="../../assets/favicon.png">
      <meta name="generator" content="mkdocs-1.6.1, mkdocs-material-9.7.2">
    
    
      
        <title>Instructions for researchers entering metadata in the catalogue - CropXR</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.484c7ddc.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.ab4e12ef.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce(((e,_)=>(e<<5)-e+_.charCodeAt(0)),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="deep-purple">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#instructions-for-researchers-entering-metadata-in-the-catalogue" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

  

<header class="md-header md-header--shadow" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="CropXR" class="md-header__button md-logo" aria-label="CropXR" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3zm0 5h18v2H3zm0 5h18v2H3z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            CropXR
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Instructions for researchers entering metadata in the catalogue
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: light)" data-md-color-scheme="default" data-md-color-primary="indigo" data-md-color-accent="deep-purple"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a4 4 0 0 0-4 4 4 4 0 0 0 4 4 4 4 0 0 0 4-4 4 4 0 0 0-4-4m0 10a6 6 0 0 1-6-6 6 6 0 0 1 6-6 6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="(prefers-color-scheme: dark)" data-md-color-scheme="slate" data-md-color-primary="deep-purple" data-md-color-accent="indigo"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 18c-.89 0-1.74-.2-2.5-.55C11.56 16.5 13 14.42 13 12s-1.44-4.5-3.5-5.45C10.26 6.2 11.11 6 12 6a6 6 0 0 1 6 6 6 6 0 0 1-6 6m8-9.31V4h-4.69L12 .69 8.69 4H4v4.69L.69 12 4 15.31V20h4.69L12 23.31 15.31 20H20v-4.69L23.31 12z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var palette=__md_get("__palette");if(palette&&palette.color){if("(prefers-color-scheme)"===palette.color.media){var media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']");palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent")}for(var[key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      
      
        <label class="md-header__button md-icon" for="__search">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        </label>
        <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.52 6.52 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
        </button>
      </nav>
      
        <div class="md-search__suggest" data-md-component="search-suggest"></div>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" tabindex="0" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
      
    
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    



  

<nav class="md-nav md-nav--primary md-nav--integrated" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="CropXR" class="md-nav__button md-logo" aria-label="CropXR" data-md-component="logo">
      
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54"/></svg>

    </a>
    CropXR
  </label>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Home
  

    
  </span>
  
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    
    
      
        
          
        
      
        
      
        
      
    
    
    
      
        
        
      
    
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <div class="md-nav__link md-nav__container">
            <a href="../" class="md-nav__link ">
              
  
  
  <span class="md-ellipsis">
    
  
    Resilience Hub
  

    
  </span>
  
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_2" id="__nav_2_label" tabindex="">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            
  
    Resilience Hub
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../defining-a-study/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Defining a study
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
        
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  
  <span class="md-ellipsis">
    
  
    Instructions for researchers entering metadata in the catalogue
  

    
  </span>
  
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  
  <span class="md-ellipsis">
    
  
    Instructions for researchers entering metadata in the catalogue
  

    
  </span>
  
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
    
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#contents" class="md-nav__link">
    <span class="md-ellipsis">
      
        Contents
      
    </span>
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
      
    
    
    
      
        
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  
  <span class="md-ellipsis">
    
  
    Policies
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            
  
    Policies
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../policies/data-sensitivity-levels/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Data sensitivity levels
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
  
    
    
      
        
          
        
      
        
      
        
      
        
      
        
      
    
    
    
      
        
        
      
    
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_4" >
        
          
          <div class="md-nav__link md-nav__container">
            <a href="../../research-drive/" class="md-nav__link ">
              
  
  
  <span class="md-ellipsis">
    
  
    Research drive
  

    
  </span>
  
  

            </a>
            
              
              <label class="md-nav__link " for="__nav_4" id="__nav_4_label" tabindex="">
                <span class="md-nav__icon md-icon"></span>
              </label>
            
          </div>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_4_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4">
            <span class="md-nav__icon md-icon"></span>
            
  
    Research drive
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../research-drive/How-to-get-access/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    How to get access
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../research-drive/Uploading-data/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Uploading data
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../research-drive/sharing-data/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Sharing data
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    
    
      
        
      
        
      
        
      
    
    
    
      
    
    
    <li class="md-nav__item md-nav__item--nested">
      
        
        
          
        
        <input class="md-nav__toggle md-toggle md-toggle--indeterminate" type="checkbox" id="__nav_4_5" >
        
          
          <label class="md-nav__link" for="__nav_4_5" id="__nav_4_5_label" tabindex="0">
            
  
  
  <span class="md-ellipsis">
    
  
    Connecting to research drive
  

    
  </span>
  
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="2" aria-labelledby="__nav_4_5_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_4_5">
            <span class="md-nav__icon md-icon"></span>
            
  
    Connecting to research drive
  

          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../research-drive/connecting-to-research-drive/Via-NextCloud/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Via NextCloud
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../research-drive/connecting-to-research-drive/Via-Rclone/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Via Rclone
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
              
                
  
  
  
  
    <li class="md-nav__item">
      <a href="../../research-drive/connecting-to-research-drive/Via-a-Browser/" class="md-nav__link">
        
  
  
  <span class="md-ellipsis">
    
  
    Via a Browser
  

    
  </span>
  
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
          
          
            <div class="md-content" data-md-component="content">
              
              <article class="md-content__inner md-typeset">
                
                  



<h1 id="instructions-for-researchers-entering-metadata-in-the-catalogue">Instructions for researchers entering metadata in the catalogue<a class="headerlink" href="#instructions-for-researchers-entering-metadata-in-the-catalogue" title="Permanent link">&para;</a></h1>
<h3 id="contents">Contents<a class="headerlink" href="#contents" title="Permanent link">&para;</a></h3>
<ul>
<li><a href="#introduction">Introduction</a></li>
<li><a href="#process-outline">Process outline</a></li>
<li><a href="#isa-fairdomseek-structure">ISA FairdomSEEK structure</a></li>
<li><a href="#miappe-and-ena-in-fairdomseek">MIAPPE and ENA in FairdomSEEK</a></li>
<li><a href="#detailed-instructions">Detailed instructions</a></li>
<li><a href="#phase-0">Phase 0</a></li>
<li><a href="#phase-1">Phase 1</a></li>
<li><a href="#phase-2">Phase 2</a> </li>
<li><a href="#phase-3">Phase 3</a></li>
<li><a href="#reference-instructions">Reference instructions</a></li>
<li><a href="#naming-conventions">Naming conventions</a></li>
<li><a href="#references-inside-sample-templates">References inside sample templates</a></li>
<li><a href="#defining-assays">Defining assays</a></li>
<li><a href="#registering-data-files">Registering data files</a></li>
<li><a href="#grouping-samples">Grouping samples</a></li>
<li><a href="#high-level-example">High level example</a></li>
<li><a href="#short-guide">Short guide</a></li>
</ul>
<h1 id="introduction">Introduction<a class="headerlink" href="#introduction" title="Permanent link">&para;</a></h1>
<p>The ResilienceHub uses FairdomSEEK with ISA settings as the metadata catalogue. This document describes how metadata can be uploaded in the right format. This document contains a detailed step by step guide for first time data entry. A shorter reference guide for additional studies will be included later.</p>
<p>There is a format to stick to, but also the format accommodates for many type of studies, so it leaves room to fill out differently. This puts some responsibility on the researcher for how to use this format and to really understand the model. The documents guide you through the choices you need to make, and how you can make them. Do plan time for this, especially the first time.  If you are stuck at any point, things are unclear or you need help, please reach out to the DataXR team at data@cropxr.org</p>
<h2 id="process-outline">Process outline<a class="headerlink" href="#process-outline" title="Permanent link">&para;</a></h2>
<p>We suggest you do phase 0 a few days before you plan to do your metadata entry and that you do step 1 and 2 in one session. </p>
<h3 id="phase-0-preparations">Phase 0: preparations<a class="headerlink" href="#phase-0-preparations" title="Permanent link">&para;</a></h3>
<p>Get an overview over the process and prepare data entry. Importantly, <strong>read this document all the way through</strong> (except the detailed step-wise instructions of phase 2, 3 and 4). Try to understand the process, the metadata model, and the concepts. You will define the study by choosing what experiments to include and make sure you are <strong>registered upfront</strong>. An admin from DataXR might need to approve requests, so make sure to do this a few days before your planned data entry. You gather files that contain metadata regarding your study.</p>
<h3 id="phase-1-define-study-assays-and-sample-templates">Phase 1: define study, assays and sample templates<a class="headerlink" href="#phase-1-define-study-assays-and-sample-templates" title="Permanent link">&para;</a></h3>
<p>In the web interface, create (an investigation and) study. Choose what fields to include in your study to properly capture the metadata. Define assays for the experiments performed in the study. </p>
<h3 id="phase-2-critical-metadata-entry">Phase 2: critical metadata entry<a class="headerlink" href="#phase-2-critical-metadata-entry" title="Permanent link">&para;</a></h3>
<p>Decide how to the metadata should be entered, to capture all information that is needed, by grouping conditions and samples. In the web interface, fill in all the fields that are essential to understand the study, focusing on key aspects, such as species and assay type. In this phase you also link the data to the performed experiments, so others can find and understand your data. </p>
<h3 id="phase-3-improving-metadata">Phase 3: improving metadata<a class="headerlink" href="#phase-3-improving-metadata" title="Permanent link">&para;</a></h3>
<p>When the fields have been defined, at any point in time you can save the metadata and edit it later. Add the additional metadata that was collected, and improves the re-usability of your data. You can also manage who can see your metadata at any time.</p>
<h2 id="isa-fairdomseek-structure">ISA FairdomSEEK structure<a class="headerlink" href="#isa-fairdomseek-structure" title="Permanent link">&para;</a></h2>
<p>The ISA format is widely used to represent research data. It is often used to represent MIAPPE data. FairdomSEEK uses the ISA structure, somewhat adapted.</p>
<h3 id="assay-and-sample-definition">Assay and sample definition<a class="headerlink" href="#assay-and-sample-definition" title="Permanent link">&para;</a></h3>
<p>In the ISA format, you can perform assays on samples. It assumes that in an experiment, there are several steps called a process. Each process has an input and an output and a protocol, that describes how the output was created from the input. The input can be either a material or data, and the output as well. These processes can be chained together. Here is an example:</p>
<div class="highlight"><pre><span></span><code>step 1 sample collection (input plant material-&gt; output sample)
step 2 DNA extraction and library prep (input sample -&gt; output sample) 
step 3 sequencing (input sample -&gt; output data)
step 4 analysis (input data -&gt; output data)
</code></pre></div>
<p>In ISA these processes are organized the following way: the study contains 1 process with a material input (study source) to a material output (study sample). The other processes, however many needed, are grouped under the assay. The assay can consist of several steps/processes and starts with the study sample.</p>
<h3 id="isa-in-fairdomseek">ISA in FairdomSEEK<a class="headerlink" href="#isa-in-fairdomseek" title="Permanent link">&para;</a></h3>
<p>In FairdomSEEK, the processes that are grouped under the assay, are all called an assay even if the input and output are both a material object, or both data files. This does not fit with the classical meaning of the word â€˜assayâ€™, but it implements the same functionality as ISA. The assays can be connected into an assay stream. </p>
<p>Each row, that can describe a sample, but also an assay performed on a sample, or a derivation of a data file, is called always a â€˜sampleâ€™ in FairdomSEEK.</p>
<h2 id="miappe-and-ena-in-fairdomseek">MIAPPE and ENA in FairdomSEEK<a class="headerlink" href="#miappe-and-ena-in-fairdomseek" title="Permanent link">&para;</a></h2>
<p>[todo: write]</p>
<p><img alt="" src="../../img/ExampleFairdomSEEKschematic.png" /></p>
<p>intro to data model</p>
<h3 id="observation-units-and-samples">Observation units and samples<a class="headerlink" href="#observation-units-and-samples" title="Permanent link">&para;</a></h3>
<p>Level where a measurement is done.</p>
<p>Whole greenhouse, per plot, plant, sample.</p>
<h1 id="detailed-instructions">Detailed instructions<a class="headerlink" href="#detailed-instructions" title="Permanent link">&para;</a></h1>
<h2 id="phase-0">Phase 0<a class="headerlink" href="#phase-0" title="Permanent link">&para;</a></h2>
<ol>
<li>
<p>Understand the basic structure of the metadata model. </p>
</li>
<li>
<p>Decide on the <a href="../defining-a-study/">what experiments to group into (the investigation and) the study</a>.</p>
</li>
<li>
<p>Choose the study type (sequencing, phenotyping or combined). Inform DataXR well up front when your study does not fit these types.</p>
</li>
<li>
<p>Make sure that you have registered your study in this <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=TVJuCSlpMECM04q0LeCIe9rk7LtjOclKu9pKlmXMf-xUMzVUVFpYVkxZTE9VNTFXTVBMSU1EN1paTC4u">form</a>. After the admin has checked the registration, you will receive a study id you need when entering the metadata, as well as storage for your experimental data. </p>
</li>
<li>
<p>Create an account for the <a href="https://catalogue.cropresilience.org">catalogue</a> and log in.
    [Write about normal or KeyCloak?]</p>
<p>Request to be added to a project. This is done by a project admin. </p>
</li>
<li>
<p>Gather the files with the metadata that you already collected. This includes e-lab journal entries, files that outline experimental conditions per plant, files that link output files to the experiment, etc.</p>
</li>
</ol>
<h2 id="phase-1">Phase 1<a class="headerlink" href="#phase-1" title="Permanent link">&para;</a></h2>
<ol>
<li>
<p>Log in to the FairdomSEEK webpage.</p>
</li>
<li>
<p>Find or create an investigation.</p>
</li>
<li>To find an investigation, click on â€œğŸ”Browseâ€ (click the menu first when using a narrow window) in the top left corner and select â€œInvestigationsâ€.  </li>
<li>
<p>If your investigation is not yet registered, click on â€œâ•Createâ€ and select â€œInvestigationsâ€. Add a Title and select a Project and click on â€œCreateâ€.</p>
</li>
<li>
<p>From within the investigation, create a new study by clicking â€œâ•Design Studyâ€ at the top. </p>
</li>
<li>Create a tile using the naming convention.</li>
<li>
<p>Choose the extended metadata type that is relevant for your study. </p>
<p>Of the extended metadata, only fill the mandatory fields for now (to be able to save). The other fields can still be entered later. Quickly skim the fields so you are aware of what is collected on study level.</p>
</li>
<li>
<p>For now, skip the fields â€œStudy positionâ€, â€œSharingâ€, â€œCreatorsâ€, â€œPublicationsâ€, and â€œDiscussion Channelsâ€. The fields â€œSharingâ€, â€œCreatorsâ€, â€œPublicationsâ€ can be reviewed and modified later.</p>
</li>
<li>
<p>At â€œDefine Sample type for Sourceâ€, select an â€œExisting templateâ€. Choose the default template â€œCropXR sourceâ€ and click â€œApplyâ€. Now review the predefined parameters that are collected for the study source. These are the fields that will be used to describe your experimental setup and conditions. If there are any missing, you can add them at the bottom by selecting â€œâ•Add new attributeâ€. Make sure that the column â€œISA Tagâ€ is set to â€œsource_characteristicâ€. Additional fields can also be added later.</p>
<p>In MIAPPE there are many suggestions for <a href="https://github.com/MIAPPE/MIAPPE/blob/master/MIAPPE_Appendix_Environment.tsv">environmental parameters</a> and <a href="https://github.com/MIAPPE/MIAPPE/blob/master/MIAPPE_Appendix_Experimental_Factor.tsv">experimental factors</a> to collect. Please reference these lists, to get a predictable field name.</p>
<p>Please do not remove any fields; this will make your study harder to find. All fields not relevant to your study can be left empty.</p>
<p><img alt="seek-adding-sample-type-field.png" src="..\img\seek-adding-sample-type-field.png" /></p>
</li>
<li>
<p>Skip â€œSOPsâ€ for now, these will be added later.</p>
</li>
<li>
<p>At â€œDefine Sample type for Sampleâ€ at select an â€œExisting templateâ€ you need to â€œChoose a templateâ€ in the drop down selection. Now select, depending on the units of measurement in your study: â€œCropXR sampleâ€ when there are samples only, â€œCropXR sample or observation unitâ€ if there are both samples in your study as well as measurements on a different level, and â€œCropXR observation unitâ€ when there are no samples. Choose the latest version of the template you need, and click â€œApplyâ€. Here you can also check the fields and add additional fields when needed, as describe for the source, while it is likely not needed.</p>
</li>
<li>
<p>Click on â€œğŸŸ¦Createâ€</p>
</li>
<li>
<p>Make a plan for how to define the Assay Streams and Assays. (read here)</p>
</li>
<li>
<p>Click on â€œâ•Design Assay Streamâ€ at the top of the study, enter a title (following the naming convention).</p>
</li>
</ol>
<p>If needed, select the â€œExtended metadataâ€ for the assay stream. Skip all other fields for now and click on â€œğŸŸ¦Createâ€.</p>
<ol>
<li>From the created Assay Stream, create an assay by clicking â€œâ•Design Assayâ€</li>
<li>Enter a title (following the naming convention)</li>
<li>For now skip the fields â€œSharingâ€, â€œCreatorsâ€, â€œSOPsâ€, â€œPublicationsâ€, â€œDocumentsâ€ and â€œChannel discussionsâ€. </li>
<li>At â€œDefine Sample type for Assayâ€ start at the â€œExisting Templatesâ€. Depending on the type of assay you are making (based on the plan made in step 4), change the â€œISA Levelâ€ drop down to â€œassay - data fileâ€ or leave it as is. Choose the latest version of the relevant template in the drop down menu and â€œApplyâ€. </li>
<li>The sample type can be expanded with additional columns that should be included as metadata. This is relevant for assays where most parameters are kept constant, but some are varied between measurements/samples. If the fields is related to the assay performed, the column â€œISA Tagâ€ should be set to â€œparameter_valueâ€, if it is about the output data file set it to â€œdata_file_characteristicâ€. </li>
<li>If needed (not included already in the assay), create the next assay of the type data file for the raw data. </li>
<li>If needed, create the next assay of the type data file for the derived data.</li>
</ol>
<p>Now you have the outline of your metadata structure. The actual metadata can be uploaded.</p>
<h2 id="phase-2">Phase 2<a class="headerlink" href="#phase-2" title="Permanent link">&para;</a></h2>
<p>Not all steps need to happen in this exact order, but some steps are dependent on each other: study sources need to be created before study samples and study samples before assay row entries. Data files need to be registered before they can be referenced.</p>
<ol>
<li>Register your data files</li>
<li>At the top menu under â€œâ•Createâ€ choose â€œData fileâ€ </li>
<li>Choose how you want to <a href="#registering-data-files">register your data</a>. Find the URL at the data location (Research Drive).</li>
<li>Under the tab â€œRemote URLâ€ past the URL of the data location and â€œğŸŸ¦Registerâ€. Do not mind a warning about the URL in a yellow box. If there is an error and the URL cannot be registered please check if the URL is correct. </li>
<li>Fill in a â€œTitleâ€, a â€œDescriptionâ€, select a â€œProjectâ€, click â€œğŸŸ¦Next, and â€œğŸŸ¦Nextâ€ </li>
<li>Select a license. The license can be adapted later if needed. At no license the consortium agreement applies to all people users that this data is shared with. </li>
<li>Adapt the sharing of the data, either on group level or per group. At a later stage, the permissions set here automatically apply to where the data is stored. For now, that is still handled separately in the research drive. </li>
<li>
<p>Skip â€œAssociated Assaysâ€ and â€œOther associated itemsâ€by clicking â€œğŸŸ¦Next, and â€œğŸŸ¦Createâ€</p>
</li>
<li>
<p>Add SOPs (Standard operating procedure) (can also be done later).
    Each step can reference a protocol that was used: creation of samples and the grouping of observation units at the study level, the assaying protocols at assay level and the data transformation steps.</p>
</li>
<li>At the top menu under â€œâ•Createâ€ choose â€œSOPâ€</li>
<li>Click â€œBrowseâ€ to upload a local file.</li>
<li>Add a â€œTitleâ€. This needs to be specific enough to find the SOP back between other SOPs from different studies.</li>
<li>Add a â€œDescriptionâ€, select a â€œProjectâ€ and a â€œLicenseâ€. Skip â€œDiscussion Channelsâ€, adapt â€œSharingâ€, skip â€œCreatorsâ€, â€œTagsâ€ and â€œAttributionsâ€</li>
<li>If the SOP is related to an assay, it can be linked under â€œExperimental assays and Modelling analysesâ€. This can also be done at the assay. </li>
<li>If a data processing step is described by a registered workflow/processing pipeline, it can be linked under â€œWorkflowsâ€. </li>
<li>
<p>Click â€œğŸŸ¦Registerâ€</p>
</li>
<li>
<p>Edit the study by going to â€œâš™ï¸Actionsâ€ in the top corner and then â€œğŸ“Edit ISA Studyâ€</p>
</li>
<li>Fill in the description</li>
<li>Fill in extended metadata. Focus on the fields most relevant to understand the study. Skip the fields that do not apply. The fields can always be revised later. </li>
<li>Under â€œSOPsâ€ select the SOP(s) that describe the sampling and the experimental design map. </li>
<li>
<p>Click â€œğŸŸ¦Updateâ€ to apply the changes.</p>
</li>
<li>
<p>Define study source samples</p>
</li>
<li>Choose <a href="#study-source">how to group/define the sources</a>.</li>
<li>Go to the â€œSources tableâ€, by clicking the tab â€œStudy designâ€ (or from the â€œSingle pageâ€ view find the â€œSources tableâ€ in the left menu). Here you find a table with the columns that you have defined. </li>
<li>Download the template by clicking the button â€œBatch download to Excelâ€. </li>
<li>In the excel, under the Samples tab fill in the data in the fields. <ol>
<li>Ignore the first two columns.</li>
<li>The Source Name is the name that will be displayed.</li>
<li>Start with the most relevant fields to understand the study and the sources used. The data can be improved on at a later point. Fill in at least the species and the experimental group. </li>
</ol>
</li>
<li>
<p>Save the file. Upload it by under â€œUpload excel spreadsheetâ€ select â€œBrowseâ€ and click â€œğŸŸ¦Uploadâ€. Now there might be an error message. Please read it carefully and adjust the data accordingly.</p>
<p>Be aware, if you upload the same excel multiple times, a new sample will be created with the same name. To check how to update existing samples, check the phase 3 instructions.</p>
</li>
<li>
<p>Define study samples</p>
</li>
<li><a href="#study-samples">Choose what type are needed</a>.</li>
<li>Go to â€œSamples tableâ€ under â€œStudy designâ€(or from the â€œSingle pageâ€ view find the â€œSamples tableâ€ in the left menu).</li>
<li>Download the template by clicking the button â€œBatch download to Excelâ€.</li>
<li>In the excel, under the Samples tab fill in the data in the fields. <ol>
<li>Ignore the first two columns.</li>
<li>Use the Input column to <a href="#sample-inputs">link to a source</a> defined in the previous step.</li>
<li>The subject_id is the name that will be displayed.</li>
<li>There is a mandatory column called protocol. The text should refer to a registered SOP.</li>
<li>Start with the most relevant fields. The data can be improved on at a later point. </li>
</ol>
</li>
<li>
<p>Save the file. Upload it by under â€œUpload excel spreadsheetâ€ select â€œBrowseâ€ and click â€œğŸŸ¦Uploadâ€. Now there might be an error message. Please read it carefully and adjust the data accordingly.</p>
</li>
<li>
<p>Fill in the assay stream extended metadata by going to  â€œâš™ï¸Actionsâ€ in the top corner and then â€œğŸ“Edit Assay Streamâ€. Focus on the fields that are most important and â€œğŸŸ¦Updateâ€ to save.</p>
</li>
<li>
<p>For each assay defined in phase 1, enter the row data, the same way as the study source and sample: download the template, fill in the data, save and upload the template.</p>
</li>
<li>For the first assay of an assay stream the input should be a study sample (so this is a sample of observation unit). For additional assays the input is an output of the previous assay. </li>
<li>To link a registered data file as file location, use the <a href="#data-files">required format</a>.</li>
<li>For the file name, use the relative path of the exact file inside the registered file location. </li>
</ol>
<h2 id="phase-3">Phase 3<a class="headerlink" href="#phase-3" title="Permanent link">&para;</a></h2>
<ol>
<li>Update the sharing permissions that each of the created elements have. Think about who should see your study. If you are collaborating with others you can give them edit permission. 
    You can edit them by navigating to the Study/Assay/DataFile/SOP, clicking â€œâš™ï¸Actionsâ€ in the top corner and then â€œğŸ”§Manage ..â€ After making the changes make sure to â€œğŸŸ¦Updateâ€</li>
</ol>
<p>Note that in the future the permission you set in SEEK on the data file will be automatically set where the data is stored, but for now the permission is still set separately within the Research Drive.</p>
<ol>
<li>
<p>Update the study and assay extended metadata to make the metadata more complete (under â€œ â€œâš™ï¸Actionsâ€ in the top corner and then â€œğŸ“Edit ..â€)</p>
</li>
<li>
<p>Updating existing samples [todo: expand]</p>
</li>
<li>make sure itâ€™s selected before downloaded, so there is no new sample with the same name</li>
<li>
<p>You can upload multiple in multiple steps, but be aware that if you re-upload samples with the same name, a new sample will be created. Only upload new samples.</p>
</li>
<li>
<p>Adding additional columns  [todo: expand] (under â€œ â€œâš™ï¸Actionsâ€ in the top corner and then â€œğŸ“Edit ..â€)</p>
</li>
</ol>
<h1 id="reference-instructions">Reference instructions<a class="headerlink" href="#reference-instructions" title="Permanent link">&para;</a></h1>
<h2 id="naming-conventions">Naming conventions<a class="headerlink" href="#naming-conventions" title="Permanent link">&para;</a></h2>
<p>[todo: finish]</p>
<p>Study:</p>
<div class="highlight"><pre><span></span><code>&lt;received study id&gt; - &lt;study name&gt;
example â€œCXRS4 - Drought response of arabidopsis after hormal treatment with PAMPâ€
</code></pre></div>
<p>Assay stream:</p>
<div class="highlight"><pre><span></span><code>&lt;received study id&gt; - &lt;descriptive assay name&gt;
</code></pre></div>
<p>Assay: </p>
<h2 id="references-inside-sample-templates">References inside sample templates<a class="headerlink" href="#references-inside-sample-templates" title="Permanent link">&para;</a></h2>
<h3 id="sample-inputs">Sample inputs<a class="headerlink" href="#sample-inputs" title="Permanent link">&para;</a></h3>
<p>Unfortunately in FairdomSEEK the names of the samples are not used as identifiers. For this reason the column where you link to samples from the previous section (marked Input) needs to be formatted like this: </p>
<div class="highlight"><pre><span></span><code>[{&quot;id&quot;=&gt;343, &quot;type&quot;=&gt;&quot;Sample&quot;, &quot;title&quot;=&gt;&quot;yeast_wgs_02&quot;}]
</code></pre></div>
<p>You can read more <a href="https://docs.seek4science.org/help/user-guide/create-sample-isajson-compliant">here</a>.</p>
<p>When you are entering this data, it might be convenient to make a column with the ids that the catalogue has generated, and one with the sample titles, and use excel formulas to create the required format, and copy the values into the upload sheet.</p>
<div class="highlight"><pre><span></span><code>=&quot;[{&quot;&quot;id&quot;&quot;=&gt;&quot;&amp;A1&amp;&quot;, &quot;&quot;type&quot;&quot;=&gt;&quot;&quot;Sample&quot;&quot;, &quot;&quot;title&quot;&quot;=&gt;&quot;&quot;&quot;&amp;B1&amp;&quot;&quot;&quot;}]&quot;
</code></pre></div>
<h3 id="data-files">Data files<a class="headerlink" href="#data-files" title="Permanent link">&para;</a></h3>
<p>To link a registered data file use the following format. </p>
<div class="highlight"><pre><span></span><code>{&quot;id&quot;=&gt;1, &quot;type&quot;=&gt;&quot;DataFile&quot;, &quot;title&quot;=&gt;&quot;File from data access layer&quot;}
</code></pre></div>
<p>Find the data file ID, by going to the data file in the interface, and checking the number at the end of the URL https://catalogue.cropresilience.org/data_files/1 </p>
<h2 id="defining-assays">Defining assays<a class="headerlink" href="#defining-assays" title="Permanent link">&para;</a></h2>
<p>There is some flexibility in how to define assays. In FairdomSEEK, an assay (called assay stream) can be split into steps (called assays) that output material or a data file.</p>
<h3 id="default-strategy">Default strategy<a class="headerlink" href="#default-strategy" title="Permanent link">&para;</a></h3>
<ol>
<li>Create one assay stream of each type of assay performed.</li>
<li>At the assay stream, choose the extended metadata type that fits the type of assay. </li>
<li>Fill in all the information that is the same for each measurement in this extended metadata fields.</li>
<li>Create inside this assay stream a single assay of the type data file. </li>
<li>Add fields for the parameters that are different per measurement.</li>
<li>Is the rows/samples of the assay to link the output files of the assay to the measured unit/plant/sample.</li>
</ol>
<h3 id="at-significant-data-processing">At significant data processing<a class="headerlink" href="#at-significant-data-processing" title="Permanent link">&para;</a></h3>
<p>Create a next assay in the same assay stream. The protocol contains the performed data processing step. The output files can be explicitly linked to the input files.</p>
<h3 id="top-level-sensors">Top level sensors<a class="headerlink" href="#top-level-sensors" title="Permanent link">&para;</a></h3>
<p>Top level environmental measurements, with no distinction between rows/samples, can be grouped together in a single assay stream. This allows for linking the data files with the specs of the measurement/instrument, without a large part of the interface being taken up by sensor measurements, that are not relevant for the understanding the study. </p>
<ol>
<li>Create assay stream for these grouped measurements. Do not choose any extended metadata. </li>
<li>Create an assay of the type observation with a material output. In this assay each row describes a type of measurement done. </li>
<li>Create a next assay of the type data file. Each row links the output file to the described measurement.</li>
</ol>
<h3 id="multiple-files-per-experiment">Multiple files per experiment<a class="headerlink" href="#multiple-files-per-experiment" title="Permanent link">&para;</a></h3>
<p>[rewrite]</p>
<p>If there are still different experimental conditions or if there are multiple data files per experiment: split into assay and data file separately. Add the different parameter on the assay, and link the files in the second data file assay</p>
<h3 id="limitation">Limitation<a class="headerlink" href="#limitation" title="Permanent link">&para;</a></h3>
<p>It is not possible to link derived data files to input files from multiple assay streams. There is no clear place to list key output artifacts, that summarize the conclusions of the study.</p>
<h2 id="registering-data-files">Registering data files<a class="headerlink" href="#registering-data-files" title="Permanent link">&para;</a></h2>
<p>You can register a URL that contains a file or a folder.</p>
<p>The advise is to register a folder that is both </p>
<div class="highlight"><pre><span></span><code> easy to navigate inside, 
containing for example only one data type and no different types of sub-folders

the same permissions apply to the whole folder
</code></pre></div>
<p>An example would be: the all raw reads of a sequencing experiment.</p>
<p>The metadata is used to link certain experiments and samples to the files. Here you can indicate what file in the registered folder contains what data.</p>
<p>At a later stage, the permission set in SEEK will be applied to the data download. If you only register the whole study as a single data file, it is not possible to apply granular permissions on who can download your data.</p>
<h2 id="grouping-samples">Grouping samples<a class="headerlink" href="#grouping-samples" title="Permanent link">&para;</a></h2>
<p>[consider moving to top section explaining the model]</p>
<h3 id="study-source">Study source<a class="headerlink" href="#study-source" title="Permanent link">&para;</a></h3>
<p>Choose the granularity that is needed for your metadata. The sources are used to specify the biological material, as well as the growth conditions. At least each experimental block needs to be defined as a separate source. If the metadata and a sample needs to be traced back to a certain plant, you might define a source per plant.</p>
<h3 id="study-samples">Study samples<a class="headerlink" href="#study-samples" title="Permanent link">&para;</a></h3>
<p>The study samples section is used both for samples, as well as for defining observation units. 
Observation units are any level, where a measurement/observation is done. How to define this will depend on the specific design of a study, and typically be the unit for which each experiment generates a file. Think about the assays and the data files when defining the observation units.</p>
<h1 id="high-level-example">High level example<a class="headerlink" href="#high-level-example" title="Permanent link">&para;</a></h1>
<h3 id="experimental-design">Experimental design<a class="headerlink" href="#experimental-design" title="Permanent link">&para;</a></h3>
<p>2 varieties (col-0, ler) and 2 watering regimes (well watered, drought) -&gt; 4 experimental groups.
Measurements:
- RNA-seq of root and and leaf samples
- Canopy coverage per experimental group
- Some general measurements from the greenhouse.</p>
<h2 id="study-general">Study general<a class="headerlink" href="#study-general" title="Permanent link">&para;</a></h2>
<p>Choose combined sequencing and phenotyping study. Add SOP with experimental design map for MIAPPE.</p>
<h2 id="study-source_1">Study source<a class="headerlink" href="#study-source_1" title="Permanent link">&para;</a></h2>
<p>Create one source per experimental group. Specify the variety and watering regime. Add additional information that is relevant.</p>
<h2 id="study-samples_1">Study samples<a class="headerlink" href="#study-samples_1" title="Permanent link">&para;</a></h2>
<p>Create samples of type â€˜sampleâ€™ for each root sample taken. The input is the source of the experimental group that the plant was grown in. Specify the plant structure/anatomical entity as root. </p>
<p>Create samples for the leave samples in the same way.</p>
<p>Create samples of type â€˜observation_unitâ€™ for each experimental group. The level is â€˜plotâ€™. The input is the source of that experimental group.</p>
<p>Create a sample of type â€˜observation_unitâ€™ that combine all sources. The level is â€˜greenhouseâ€™. </p>
<h2 id="assays">Assays<a class="headerlink" href="#assays" title="Permanent link">&para;</a></h2>
<p>Create one phenotyping assay stream for canopy coverage and fill in all the information about the trait and the method. Create an assay with as output a data file. Create a row for each separate measurement/experimental group, with as input the observation unit plots.  Describe the output data file of the measurement and link to the data. </p>
<p>Create one sequencing assay stream. Describe the library construction and sequencing. Create an assay with as output a data file. Create a row for each measurement, with as input a sample. Describe the output data file of the measurement and link to the data. </p>
<p>Create one phenotyping assay stream for the greenhouse sensors. Create an assay for the measurements of the output material with the template observation. Define one row per environment variable measured by the greenhouse sensors, describing the variable, the sensor and the scale. This can be one row for temperature and one for humidity. The input is the study observation unit greenhouse. Create a next assay in the stream for the data files, one row for each file, with as input the assay row defined in the previous assay. Describe the and link to the data. </p>
<h1 id="short-guide">Short guide<a class="headerlink" href="#short-guide" title="Permanent link">&para;</a></h1>
<p>Prepare</p>
<ol>
<li>register the study in the form and get storage </li>
<li>gather relevant documents, such as lab notes and experimental design overviews</li>
</ol>
<p>Start with the metadata entry in FairdomSEEK</p>
<ol>
<li>create a study</li>
<li>choose an extended metadata type</li>
<li>set the sharing of the study</li>
<li>choose a template for the study source. Add additional fields if needed.</li>
<li>choose a template for the study sample (sample only, observation unit, or combined)</li>
<li>for each assay type create an assay stream</li>
<li>select the right extended metadata</li>
<li>set the sharing</li>
<li>within the assay stream create one or more assays </li>
<li>choose a sample template that fits the type of assay (choose between material and data file + choose the full or condensed + choose sequencing or observation)</li>
<li>add additional fields if needed</li>
<li>add additional assays for data files that were generate from the first assay output</li>
</ol>
<p>Now the study has been defined, continue with the minimally needed metadata.</p>
<ol>
<li>Register Data files by linking the URL to the file or folder. Manage the permissions on the file. </li>
<li>Register protocols under SOPs and manage the permissions. </li>
<li>Edit the study to fill the extend metadata and link SOPs.</li>
<li>Define sources, based on your specific study and the level of granularity needed.</li>
</ol>
<p>Enter the data by downloading the template, filling the data and uploading.
10. Define samples/observation units, based on your specific study and the level of granularity needed.</p>
<div class="highlight"><pre><span></span><code>Enter the data by downloading the template, filling the data, referencing the study sources by name and id, and uploading the template in the interface.
</code></pre></div>
<ol>
<li>Edit assay metadata </li>
<li>Enter assay row data + link data files</li>
</ol>
<p>Now that the minimal metadata is added, continue improving the metadata.</p>
<ol>
<li>adapt permissions </li>
<li>update with additional data</li>
</ol>












                
              </article>
            </div>
          
          
  <script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var labels=set.querySelector(".tabbed-labels");for(var tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>

<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
      <div class="md-consent" data-md-component="consent" id="__consent" hidden>
        <div class="md-consent__overlay"></div>
        <aside class="md-consent__inner">
          <form class="md-consent__form md-grid md-typeset" name="consent">
            



<h4>Cookie consent</h4>
<p>We use cookies to recognize your repeated visits and preferences, as well as to measure the effectiveness of our documentation and whether users find what they're searching for. With your consent, you're helping us to make our documentation better.</p>
<input class="md-toggle" type="checkbox" id="__settings" >
<div class="md-consent__settings">
  <ul class="task-list">
    
    
    
      
    
    
      
        
  
  
    
    
      
    
  
  <li class="task-list-item">
    <label class="task-list-control">
      <input type="checkbox" name="analytics" checked>
      <span class="task-list-indicator"></span>
      Google Analytics
    </label>
  </li>

      
    
  </ul>
</div>
<div class="md-consent__controls">
  
    
      <button class="md-button md-button--primary">Accept</button>
    
    
    
  
    
    
      <button type="reset" class="md-button md-button--primary">Reject</button>
    
    
  
    
    
    
      <label class="md-button" for="__settings">Manage settings</label>
    
  
</div>
          </form>
        </aside>
      </div>
      <script>var consent=__md_get("__consent");if(consent)for(var input of document.forms.consent.elements)input.name&&(input.checked=consent[input.name]||!1);else"file:"!==location.protocol&&setTimeout((function(){document.querySelector("[data-md-component=consent]").hidden=!1}),250);var form=document.forms.consent;for(var action of["submit","reset"])form.addEventListener(action,(function(e){if(e.preventDefault(),"reset"===e.type)for(var n of document.forms.consent.elements)n.name&&(n.checked=!1);__md_set("__consent",Object.fromEntries(Array.from(new FormData(form).keys()).map((function(e){return[e,!0]})))),location.hash="",location.reload()}))</script>
    
    
      
      
      <script id="__config" type="application/json">{"annotate": null, "base": "../..", "features": ["navigation.instant", "navigation.tracking", "navigation.sections", "navigation.expand", "navigation.indexes", "navigation.top", "toc.integrate", "search.suggest", "search.highlight", "content.tabs.link", "content.code.copy", "content.code.annotate"], "search": "../../assets/javascripts/workers/search.2c215733.min.js", "tags": null, "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}, "version": null}</script>
    
    
      <script src="../../assets/javascripts/bundle.79ae519e.min.js"></script>
      
    
  </body>
</html>